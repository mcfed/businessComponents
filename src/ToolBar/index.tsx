import React, {FunctionComponent} from 'react';

interface IToolBarProps {
  /**
   * 样式名称
   */
  className?: string;
}

const ToolBar: FunctionComponent<IToolBarProps> = props => {
  const {children, className = ''} = props;
  return <div className={`tool-bar ${className}`}>{children}</div>;
};

export default ToolBar;
