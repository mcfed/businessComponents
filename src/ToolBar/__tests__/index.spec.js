import React from 'react';
import {render} from 'enzyme';
import {Button} from 'antd';
import ToolBar from '../index';

describe('toolbar base test', () => {
  it('test render', () => {
    const wrapper = render(
      <ToolBar>
        <Button>1</Button>
        <Button>2</Button>
      </ToolBar>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
