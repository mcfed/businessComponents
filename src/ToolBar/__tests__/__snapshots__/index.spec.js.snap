// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toolbar base test test render 1`] = `
<div
  class="tool-bar "
>
  <button
    class="ant-btn"
    type="button"
  >
    <span>
      1
    </span>
  </button>
  <button
    class="ant-btn"
    type="button"
  >
    <span>
      2
    </span>
  </button>
</div>
`;
